fldigi.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.freedesktop.fldigi
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: fldigi
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
```

Comment:
What does it needs home access for?

Reply:
in the user's home folder it keeps a logbook of the contacts made
user must be able to send text from file
you think it is not necessary can be removed
it's my first flatpak, have me passience